Extensions/Build/Symbols/Symbols.dnn

```diff
@@ -1,6 +1,6 @@
-﻿<dotnetnuke type="Package" version="5.0">
+<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="Dnn.PersonaBar.Extensions_Symbols" type="Library" version="07.01.02">
+    <package name="Dnn.PersonaBar.Extensions_Symbols" type="Library" version="01.06.01">
```

Comment:
Similar comment on version number here

Reply:
Was the symbol package ever published? We don't have any published releases of the personabar, and the symbols don't appear to be on any previous release of platform... right? Or am I missing something? They'll be one more commit pushed to update the nuspec file version nodes in a few minutes....